<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialNetworkUI</class>
 <widget class="QMainWindow" name="SocialNetworkUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocialNetworkUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="loginWidget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>login</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="loginEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="loginLable">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter your name.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="profileWidget" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>471</width>
      <height>511</height>
     </rect>
    </property>
    <widget class="QLabel" name="headerLable">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>my profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="followButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>add friend</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="postEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="postBrowser">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>60</y>
       <width>251</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="postButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>290</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>post</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="suggestTable">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>330</y>
       <width>141</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="friendsTable">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>300</y>
       <width>251</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
